### PR TITLE
Add Hardware Sentry TrueSight Presentation connector

### PR DESCRIPTION
**Pull request by SanduDS**
## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: openapi/bmc.truesightpresentationserver/client.bal [hunk 1]
```diff
@@ -0,0 +1,428 @@
...
+    final readonly & map<string> apiKeys;
+    # Gets invoked to initialize the `connector`.
+    # The connector initialization requires setting the API credentials.
+    # Create a [user account](https://docs.bmc.com/docs/sso90/managing-users-474056900.html#Managingusers-Toaddanewuser) in BMC Hardware Sentry TrueSight Presentation Server and obtain tokens following [this guide](https://docs.bmc.com/docs/display/tsps107/TSWS+authentication).
+    #
+    # + apiKeyConfig - Provide your API Key as `Authorization`. Eg: {Authorization: authToken [<API_KEY>]} 
+    # + clientConfig - The configurations to be used when initializing the `connector` 
+    # + serviceUrl - URL of the target service 
+    # + return - An error if connector initialization failed 
+    public isolated function init(ApiKeysConfig apiKeyConfig, http:ClientConfiguration clientConfig =  {}, string serviceUrl = "https://localhost:8043/tsws/10.0/api/") returns error? {
```
Comment (Chuhaa) [inline openapi/bmc.truesightpresentationserver/client.bal:40]: It points a localhost url. So shall we remove `server` in openapi?

Comment (SanduDS) [reply openapi/bmc.truesightpresentationserver/client.bal:40]: Removing the default link?

Comment (Chuhaa) [reply openapi/bmc.truesightpresentationserver/client.bal:40]: yes. This is not a valid service url

Comment (SanduDS) [reply openapi/bmc.truesightpresentationserver/client.bal]: Fixed removing default openAPI server tag

**Review by indikasampath2000 — APPROVED** [review]

**Pull request merged**